apps/site/src/components/PageTransition/PageTransition.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
```

Comment:
Concern: Hesitant to sacrifice Server Components for this design when this becomes part of the parent layout making all children Client Components.

Reply:
working on a way to avoid wrapping everything from layout.tsx